python/dronin/uavo.py

```diff
@@ -442,6 +442,10 @@ def make_class(collection, xml_file, update_globals=True):
                     values = tuple(float(v) for v in info['defaultvalue'].split(','))
                     if info['type'] != 'float':
                         values = tuple(int(v) for v in values)
+                    else:
```

Comment:
The "onlydefault" mode was returning every field that has a default value that can't be represented in single precision precisely, because we had doubles here.  Round trip through a float here to get the default values as single precision.